src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1661,9 +1661,14 @@ public String toString() {
 			result.append(", height: ");
 			result.append(getHeight());
 			result.append(", frame rate: ");
+			if (isNotBlank(frameRate)) {
 			result.append(getFrameRate());
+			} else { result.append("variable"):
+			}
```

Comment:
I was sure you'll say that ;)

About `(videoBitDepth != 8)`i'm not sure if that the best way to code it, though many in UMS code; Should the use of `equals()` method be preferred ?